File: QuestoesBeecrowd-AD_HOC/1329.c
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>
#include <math.h>
#include <string.h>

int main()
{
      setlocale(LC_ALL, "");
      int i, q, e, f = 1, m = 0, j = 0;
      while (f == 1)
      {

            scanf("%d", &q);
            switch (q)
            {
            case 0:
                  return 0;
                  

            default:
                  for (i = 0; i < q; i++)
                  {
                        scanf("%d", &e);
                        switch (e)
                        {
                        case 0:
                              m = m + 1;
                              break;
                        case 1:
                              j = j + 1;
                              break;
                        default:
                              break;
                        
                        }
                  }
                  printf("Mary won %d times and John won %d times\n", m, j);
                  m = 0, j = 0;
                  break;
            }
      }

      return 0;
}

File: QuestoesBeecrowd-AD_HOC/1329.java
import java.math.BigInteger;
import java.util.Scanner;
import java.lang.*;
import java.util.Locale;

public class Main {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int i, q, e, f = 1, m = 0, j = 0;
        int retu = 0;
        while (f == 1) {

            q = input.nextInt();
            switch (q) {
                case 0:

                    System.exit(0);

                default:
                    for (i = 0; i < q; i++) {

                        e = input.nextInt();
                        switch (e) {
                            case 0:
                                m = m + 1;
                                break;
                            case 1:
                                j = j + 1;
                                break;
                            default:
                                break;

                        }
                    }
                    System.out.println("Mary won " + m + " times and John won " + j + " times");

                    m = 0;
                    j = 0;
                    break;
            }
        }

    }

}

